testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -161,11 +162,13 @@ def test_volume(self):
 class TestGROWriter(TestCase, tempdir.TempDir):
     def setUp(self):
         self.universe = mda.Universe(GRO)
+        self.large_universe = mda.Universe(GRO_large)
         self.prec = 2  # 3 decimals in file in nm but MDAnalysis is in A
         ext = ".gro"
         self.tmpdir = tempdir.TempDir()
         self.outfile = self.tmpdir.name + '/gro-writer' + ext
         self.outfile2 = self.tmpdir.name + '/gro-writer2' + ext
+        self.outfile3 = self.tmpdir.name + '/gro-writer3' + ext
```

Comment:
Is there a reason why the output path for all tests in the class are set at the class level in the `setUp` and not in the corresponding test? (Note that it was obviously like that before the PR already.)